code/game/supplyshuttle.dm

```diff
@@ -1331,9 +1331,9 @@ var/datum/controller/supply/supply_controller = new()
 	. = ..()
 
 	vehicles = list(
-		/datum/vehicle_order/apc,
-		/datum/vehicle_order/apc/med,
-		/datum/vehicle_order/apc/cmd,
+		new /datum/vehicle_order/apc(),
+		new /datum/vehicle_order/apc/med(),
+		new /datum/vehicle_order/apc/cmd(),
```

Comment:
If you instantiate these here you can't also do it below as this causes runtimes on mapload.

https://github.com/cmss13-devs/cmss13/blob/a17e6a58a761327effedff67f38b7b597231ad65/code/game/supplyshuttle.dm#L1339-L1340